proposals/0187-sbpf-stricter-elf-headers.md

```diff
@@ -0,0 +1,131 @@
+---
+simd: '0187'
```

Comment:
Can you update the SIMD number (and PR title) to match the PR number, per https://github.com/solana-foundation/solana-improvement-documents/blob/main/proposals/0001-simd-process.md#draft